instant-xml-macros/src/meta.rs

```diff
@@ -128,10 +128,20 @@ impl NamespaceMeta {
                     }
 
                     NsState::PrefixValue {
-                        prefix: segment.ident,
+                        prefix: segment.ident.to_string(),
                     }
                 }
-                (NsState::Prefix, TokenTree::Ident(id)) => NsState::Eq { prefix: id },
+                (NsState::Prefix, TokenTree::Ident(id)) => NsState::Eq {
+                    prefix: id.to_string(),
+                },
+                (NsState::Eq { mut prefix }, TokenTree::Punct(punct)) if punct.as_char() == '-' => {
```

Comment:
I think this should allow `_` and `.` as well.

I'm not sure if there is value in modelling the constraints as well but I believe the first character has to be ascii or `_` and the string cannot contain `xml` or `:`.

Reply:
Added support for `.` as well. The code is matching on an Identifier token in the token stream which will be a little more lenient than what you're describing, it should already fail on `:` (by virtue of the parser failing closed) and I think it's probably okay not to model the `xml` constraint to allow Rust types to model elements/attributes that are defined by XML standards.